controllers/managedocs_controller.go

```diff
@@ -835,6 +836,10 @@ func (r *ManagedOCSReconciler) reconcileAlertmanagerConfig() error {
 		if smtpPassword == "" {
 			return fmt.Errorf("smtp secret does not contain a password entry")
 		}
+		smtpHTML, err := ioutil.ReadFile("customernotification.html")
```

Comment:
How does `ReadFile` know to search at the `templates` subdirectory?

Reply:
Ok

Reply:
I have made the change, can you pls re-review it